### DISCUSSION TITLE
How dangerous is this for rclone mounted files?

### DISCUSSION THREAD

**Discussion by dannymichel** [ptr727/PlexCleaner#117] [General]
How dangerous is this for rclone mounted libraries and 200tb libraries?

**Comment by ptr727** [comment]
If you quantify "dangerous" as file corruption or deletion, I don't know.
I don't use or test rclone mounted libs myself, so I'd suggest you test until you are comfortable with the results.
E.g. create a remote directory with test files that represent your library, and run `process` against that.

**Comment by Anon0511** [comment]
Some months have passed, have you or someone else tested it?

**Reply by dannymichel** [reply]
Too scared to be honest

**Reply by Anon0511** [reply]
lmao me too
I'll think about it

**Reply by Anon0511** [reply]
I just started it on a tiny library using the fuse-mounts of my media. That causes all files to be moved to the local storage first and processing them there. This is probably the cleanest way to do it, since it is not going towards your Clouds API hits if they're restricted, but you should probably avoid using --parallel **or** have enough storage to keep up with the speed of transcoding when uploading them back to the cloud. When uploading, it'd make sense to exclude .PlexCleaner and .tmp* files. Anyway I see this causing issues and trouble on any library being larger than your local storage because the original file won't have any flag added and would get uploaded too. 

So I assume the best way to avoid that would be to use the pure rclone mount folder, but I have no idea how happy your Cloud provider and the API limits will be about it. It'll also get a significant hit in speed I presume! I do have Google and I'll try to check, but since I do have some things running that splits API usage and contingents over 10 projects it'll probably be difficult to determine. But this is the next thing I'll try with another smaller library.

**Reply by Anon0511** [reply]
And that's the current resource usage on the system 
![image](https://user-images.githubusercontent.com/30687744/193897457-b3e73145-574b-4936-8a81-8ecff8203cfc.png)
